ddtrace/profiling/profiler.py

```diff
@@ -221,6 +224,9 @@ def __attrs_post_init__(self):
                 recorder=r, exporters=exporters, before_flush=self._collectors_snapshot
             )
 
+        if self.asyncio_loop_policy is not None:
+            _asyncio.asyncio.set_event_loop_policy(self.asyncio_loop_policy)
```

Comment:
how do people use event loop policies? will this have any conflicts with other applications or user settings?

(probably something I need to go read more about)

Reply:
From my gathering, you don't use it except if you have a good reason to choose another policy other than the default one. There's only one default policy per OS in CPython, so there's nothing else to choose from.

But you're right, if somebody decides to use [uvloop](https://github.com/MagicStack/uvloop) instead, that'd not work anymore.

We probably need to document that part.

Reply:
If supporting evidence helps, `Sanic` will activate uvloop by default.